python/Ganga/Utility/feedback_report.py

```diff
@@ -652,6 +653,15 @@ def renameDataFiles(directory):
                 logger.debug("Err %s" % str(err))
                 writeErrorLog(str(sys.exc_info()[1]))
 
+        # Copy thread stack trace file
+        try:
+            thread_trace_source_path = os.path.join(getConfig('Configuration')['gangadir'], thread_trace_file_name)
+            thread_trace_target_path = os.path.join(fullLogDirName, thread_trace_file_name)
+            shutil.copyfile(thread_trace_source_path, thread_trace_target_path)
```

Comment:
It's not. It's:

1. Setting `thread_trace_source_path` to the source of the file (`~/.gangadir/thread_trace.html`) where the background thread is writing it.
2. Setting `thread_trace_target_path` to where the file should be copied to.
3. Copying the file at `thread_trace_source_path` to `thread_trace_target_path`

Unless I misunderstand you or have made a mistake with this code?